### ISSUE TITLE
[IDP] [SB] Forgot password is not working for study builder admin in the following scenario

### ISSUE THREAD

**Issue by bhoomikav16** [GoogleCloudPlatform/fda-mystudies#4659]
**Pre-condition:** IDP should be disabled for the SB

**Steps:**
1. Click on 'Forgot password' link 
2. Enter the Email which was added as organizational admin 
3. Click on 'Submit' and Verify

**AR:** Forgot password is not working for study builder admin in the following scenario
**ER:** Admin should be able change the password 

![sb](https://user-images.githubusercontent.com/86007179/176426614-5bed2a12-3bce-4d43-ac4f-56ccd7a4ef84.png)

**Comment by harisboston** [comment]
@bhoomikav16 As per initial discussion with BA, the above error should display for organizational users.
This user was added when IdP was enabled, after we disabled IdP, still its Org. user so the app is showing an error message.
The Above error message will not show for non-org users.
Super admin should delete that user and again invite that user from SB when Idp is disabled to make it a non-org user.
